polar-core/src/debugger.rs

```diff
@@ -65,6 +65,7 @@ enum Step {
 /// [`Goal::Query`](../vm/enum.Goal.html). When either breakpoint is hit, we check the
 /// [`Debugger`](struct.Debugger.html)'s internal [`step`](struct.Debugger.html#structfield.step)
 /// field to determine how evaluation should proceed.
+#[allow(clippy::large_enum_variant)]
```

Comment:
Is this needed?

Reply:
yep the introduction of our `MissingRequiredRule { Rule }` struct caused this to balloon and clippy complain.